src/server/main_service.cc

```diff
@@ -167,6 +169,9 @@ std::string MakeMonitorMessage(const ConnectionState& conn_state,
 }
 
 void DispatchMonitorIfNeeded(bool admin_cmd, ConnectionContext* connection, CmdArgList args) {
+  if (connection->journal_emulated && !GetFlag(FLAGS_debug_replica_monitor)) {
```

Comment:
why using a flag debug_replica_monitor and not use the Monitor command in replica

Reply:
I do it on the replica already. I was not sure the replica reports replication commands by default, but I just checked, it does, so I'll remove it